apisix/plugins/sls-logger.lua

```diff
@@ -61,15 +61,28 @@ local schema = {
     required = {"host", "port", "project", "logstore", "access_key_id", "access_key_secret"}
 }
 
+local metadata_schema = {
```

Comment:
What is the relationship between this PR and sls-logger?
Will this fix cause the sls-logger ci-test to fail?

Reply:
Yes, sls-logger doesn't have a metadata schema, but the test case does.
https://github.com/apache/apisix/blob/07c4aa3a4287b4a829b76ae529705b51de56f8aa/t/plugin/sls-logger.t#L209-L222
because it doesn't go to the actual calibration logic so it doesn't report an error.

After adding the id, the check here will fail https://github.com/apache/apisix/blob/6edd86313b800e79cb53a8bb10c3908a7f5348fe/apisix/plugin.lua#L1041-L1042